.env.example

```
@@ -4,3 +4,16 @@
 DB_URL=
 DB_USERNAME=quespot
 DB_PASSWORD=
+
+REDIS_HOST=localhost
+REDIS_PORT=6379
+REDIS_PASSWORD=
+
+MAIL_USERNAME=
+MAIL_PASSWORD=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MAIL_PASSWORD key should go before the MAIL_USERNAME key

(UnorderedKey)